### PR TITLE
"[All Hosts] (manifest) add req set and verionoverrides schema info to VersionOverrides and some children

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
This is another in a series of PRs to add this kind of info to the VersionOverrides element and its descendants. This one also splits the VersionOverrides ref topic into an overview and 4 separate topics for each of the schemas it is part of.

### CODE REVIEW

File: docs/reference/manifest/versionoverrides-1-0-content.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: VersionOverrides 1.0 element in the manifest file for a content add-in
+description: 'Reference documentation of the VersionOverrides element (content) for Office Add-ins manifest (XML) files.'
```
Comment (alison-mk) [inline docs/reference/manifest/versionoverrides-1-0-content.md:3]: I'm curious if our editor @lindalu-MSFT has any suggestions for the description fields of these new articles. The descriptions are very similar, and I find the distinction between them unclear. Perhaps Linda could help clarify.

**Review by alison-mk — APPROVED** [review]
I approve with two caveats: 
1. I suggest also getting an editor review from @lindalu-MSFT, particularly for the meta description fields. 
2. I left one question + suggestion about a sentence that I find confusing.

File: docs/reference/manifest/versionoverrides-1-0-mail.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+|  [Hosts](hosts.md)                |  Yes  |  Specifies a collection of Office applications. The child Hosts element overrides the Hosts element in the parent portion of the manifest.  |
+|  [Resources](resources.md)    |  Yes  | Defines a collection of resources (strings, URLs, and images) that other manifest elements reference.|
+|  **VersionOverrides**    |  No  | Defines add-in commands under a newer schema version. See [Implementing multiple versions](#implementing-multiple-versions) for details. |
+|  [WebApplicationInfo](webapplicationinfo.md)    |  No  | Specifies details about the add-in's registration with secure token issuers, such as Azure Active Directory V2.0. |
+
+### Description
+
+Describes the add-in. This overrides the `Description` element in any parent portion of the manifest. The text of the description is contained in a child element of the **LongString** element contained in the [Resources](resources.md) element. The `resid` attribute of the **Description** element can be no more than 32 characters and is set to the value of the `id` attribute of the `String` element that contains the text.
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/reference/manifest/versionoverrides-1-0-mail.md:46]:
```suggestion
Describes the add-in. This overrides the `Description` element in any parent portion of the manifest. The text of the description is contained in a child element of the **LongString** element contained in the [Resources](resources.md) element. The `resid` attribute of the **Description** element can be no more than 32 characters and must match the value of the `id` attribute of a child element of the **ShortString** element contained in the [Resources](resources.md) element. 
```

**Comment by lindalu-MSFT** [comment]
I made a couple comments.

  1.  The metadata descriptions are all fine, except some are surrounded by a single quote and some are not. Not sure why the inconsistency?
  2.  You don’t use contractions. I.e., you are, do not, etc.
  3.  Better word choice for the word “fuller” as in “for fuller examples”. I suggest changing to “more complex examples”. Fuller doesn’t necessarily mean more complex so I think it’s a better word choice for localization.

Also, I misspoke previously. I do get a ping when you add me as a reviewer! My fault that I overlooked it. Keep in mind that I really DO try hard to get to your editorial needs as quickly as I can so if you don’t hear from me in say, 48 hours, ping me again! I try to do it same or next day, unless it’s a massive edit that I need to break up into chunks.

Thanks,
Linda

**Pull request merged**